CHANGELOG-0.7.0.md

```diff
@@ -0,0 +1,13 @@
+# Changelog since v0.6.1
```

Comment:
Also can you rename the filename to the minor release? CHANGELOG-0.7.md

Reply:
Done